src/validators/__schema.ts

```diff
@@ -0,0 +1,20 @@
+import type { SomeSchema, DefaultModifiers } from '../types';
+
+export const TYPEOFWEB_SCHEMA = Symbol('@typeofweb/schema');
+
+export const isSchema = (val: any): val is SomeSchema<any> => {
+  return (
+    typeof val === 'object' &&
+    val !== null &&
+    '__validator' in val &&
+    '__modifiers' in val &&
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    val[TYPEOFWEB_SCHEMA] === true
```

Comment:
Why do we have  to check ourselves? If `TYPEOFWEB_SCHEMA` is unique and only we can set this property. If we set this property to true, then isn't val always an object, `'__validator'` always on this object and `'__modifiers'` also always on this object?